website/source/docs/agent/acl-system.html.md

```diff
@@ -0,0 +1,195 @@
+ many tokens.
+
+ * **ACL Tokens** - Requests to Consul are authorized by using bearer token. Each ACL token has a public
+ Accessor ID which is used to name a token, and a Secret ID which is used as the bearer token used to
+ make requests to Consul.
+
+ ACL tokens and policies are managed by Consul operators via Consul's
+[ACL API](/api/acl/acl.html), ACL CLI, or systems like
+[HashiCorp's Vault](https://www.vaultproject.io/docs/secrets/consul/index.html).
+
+### ACL Policies
+
+An ACL policy is a named set of rules and is composed of the following elements:
+
+* **ID** - The policies auto-generated public identifier.
+* **Name** - A unique meaningful name for the policy.
+* **Rules** - Set of rules granting or denying permissions. See the [Rule Specification](#rule-specification) section for more details.
+* **Datacenters** - A list of datacenters the policy is valid within.
+
+#### Builtin Policies
+
+* **Global Management** - Grants unrestricted privileges to any token that uses it. When created it will be named `global-management`
+and will be assigned the reserved ID of `00000000-0000-0000-0000-000000000001`. This policy can be renamed but modification
+of anything else including the rule set and datacenter scoping will be prevented by Consul.
+
+### ACL Tokens
+
+ACL tokens are used to determine if the caller is authorized to perform an action. An ACL token is composed of the following
+elements:
+
+* **Accessor ID** - The token's public identifier.
+* **Secret ID** -The bearer token used when making requests to Consul.
+* **Description** - A human readable description of the token. (Optional)
+* **Policy Set** - The list of policies that are applicable for the token.
+* **Locality** - Indicates whether the token should be local to the datacenter it was created within or created in
+the primary datacenter and globally replicated.
+
+#### Builtin Tokens
+
+During cluster bootstrapping when ACLs are enabled both the special `anonymous` and the `master` token will be
+injected.
+
+* **Anonymous Token** - The anonymous token is used when a request is made to Consul without specifying a bearer token.
+The anonymous token's description and policies may be updated but Consul will prevent this tokens deletion. When created,
+it will be assigned `00000000-0000-0000-0000-000000000002` for its Accessor ID and `anonymous` for its Secret ID.
+
+* **Master Token** - When a master token is present within the Consul configuration, it is created and will be linked
+With the builtin Global Management policy giving it unrestricted privileges. The master token is created with the Secret ID
+set to the value of the configuration entry.
+
+#### Authorization
+
+The token Secret ID is passed along with each RPC request to the servers. Consul's
+[HTTP endpoints](/api/index.html) can accept tokens via the `token`
+query string parameter, the `X-Consul-Token` request header, or Authorization Bearer
+token [RFC6750](https://tools.ietf.org/html/rfc6750). Consul's
```

Comment:
I would probably word this like: "or an RFC6750 authorization bearer token"

Having RFC6750 at the end doesn't make much sense (although its possible I wrote it like this initially)

Reply:
updated